plugins/commands.py

```diff
@@ -1,44 +1,31 @@
-"""commands which starts with slash /
+from pyrogram import Client, filters
+from pyrogram.types import Message
 
-# ../config
-COMMANDS_CHATS = MUSIC_CHATS
-COMMANDS_TEXT_START = (
-    "This is a music downloader bot for "
-    "members of the channel and group"
-)
-COMMANDS_TEXT_CONTACTS = (
-    "Regarding any issues with the bot "
-    "feel free to contact"
-)
 COMMANDS_TEXT_HELP = (
-    MUSIC_INFORM_AVAILABILITY
-    + "\n\n<b>Usage</b>:\n"
+    "This bot only serves cpop.tw and "
+    "its members in private chat"
+    "\n\n<b>Usage</b>:\n"
     "- Send a message that only contains a YouTube/SoundCloud/Mixcloud link "
     "to download the music\n"
     "- Playlists are not supported\n"
     "- Your message will be deleted in private chat after the music gets "
     "successfully uploaded\n"
     "- You can get YouTube links with inline bot @vid\n\n"
-    + COMMANDS_TEXT_CONTACTS
+    "Regarding any issues with the bot "
+    "feel free to contact @konnov"
 )
 
-"""
-from pyrogram import Client, filters
-from pyrogram.types import Message
-from config import COMMANDS_CHATS, COMMANDS_TEXT_START, COMMANDS_TEXT_HELP
-
 
 @Client.on_message(filters.command(["start"])
-                   & (filters.chat(COMMANDS_CHATS) | filters.private)
                    & filters.incoming
                    & ~filters.edited)
 async def command_start(_, message: Message):
     """/start introduction of the bot"""
-    await message.reply(COMMANDS_TEXT_START)
+    sticker = open('static/hello_animated_sticker.tgs', 'rb')
+    await message.reply_sticker(sticker)
 
```

Comment:
You forgot to `.close()` the file, and you can just pass the filename to `reply_sticker()` directly.

`await message.reply_sticker("static/hello_animated_sticker.tgs")`

> sticker (str) – Sticker to send. Pass a file_id as string to send a sticker that exists on the Telegram servers, pass an HTTP URL as a string for Telegram to get a .webp sticker file from the Internet, or pass a file path as string to upload a new sticker that exists on your local machine.

btw I think that it's bad to upload a file every time a user `/start` the bot. for images it's better to just pass a URL with correct mime types. But it does not work for me with `.tgs`. and other possible workarounds for avoiding reupload does not worth the effort.